src/NodeProcess.js

```diff
@@ -56,6 +56,15 @@ NodeProcess.prototype.store = T.func([T.String], T.Nil, "nodeProcess.store").of(
 });
 
 /**
+ * Returns the last process output
+ */
+NodeProcess.prototype.lastOutput = T.func([], T.Object, "nodeProcess.lastOutput").of(function() {
+    return assign({}, {
+        lastOutput: this.instance.output.last
```

Comment:
``
import { last } from "lodash";

lastOutput: last(this.instance.output)
``